libs/jwst-codec/src/doc/codec/content.rs

```diff
@@ -212,36 +203,25 @@ impl Content {
             Self::String(string) => {
                 encoder.write_var_string(string)?;
             }
-            Self::Embed(json) => {
-                encoder.write_var_string(json.to_string())?;
+            Self::Embed(val) => {
+                encoder.write_var_string(serde_json::to_string(val).unwrap())?;
```

Comment:
Don't `unwrap` unless absolutely necessary, you can use `map_err` here or implement `From<serde_json::Error>` for `JwstError`.